aries-site/src/examples/components/button/DosDonts/ButtonGoodCancelPreview.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Heading, Header, Text } from 'grommet';
+
+export const ButtonGoodCancelPreview = () => (
+  <Box width="large" gap="medium" pad="small">
+    <Header
+      direction="column"
+      align="start"
+      gap="xxsmall"
+      pad={{ horizontal: 'xxsmall' }}
+    >
+      <Heading level={2} margin="none">
```

Comment:
```suggestion
      <Heading level={2} margin="none" size="small">
```
In most designs from teams, I've seen use of size small.

Reply:
sounds good